src/context.rs

```diff
@@ -639,7 +652,10 @@ pub struct InteractionState {
     pub(crate) using_keyboard_navigation: bool,
 }
 
+/// Holds current layout state for given position in the tree.
+/// You'll use this in the `View::layout` implementation to call `layout_node` on children and to access any font
 pub struct LayoutCx<'a> {
+    // TODO: seems like custom widgets like scroll, tab, text_input and more need access to this. Should we make this a public method on LayoutCx?
```

Comment:
seems like custom widgets like scroll, tab, text_input and more need access to this. Should we make this a public method on LayoutCx?

Reply:
Maybe ```Deref``` ```LayoutCx``` to ```AppState```?

Reply:
I like that, I'll do that in a separate PR.